extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/PrivilegesBuilder.scala

```diff
@@ -593,6 +594,9 @@ object PrivilegesBuilder {
           Nil,
           owner)
 
+      case "SetTableOwnerCommand" =>
```

Comment:
It's a command from auth. For better self alignment and readability, shall we use `classOf[SetTableOwnerCommand].getSimpleName` instead

Reply:
Done